server/models/index.js

```diff
@@ -10,6 +10,7 @@ module.exports = {
           callback(null, results);
         },
         (err) => {
+	  console.log('getRelated:', err);
```

Comment:
Do you want this console log to persist in prod?

Reply:
Yes. It only shows if you are logged in via SSH running it manually instead of letting the daemon do it.